static/styles/components.css

```diff
@@ -160,11 +160,22 @@
     width: 90px;
     margin: 0px -0.5px;
     text-align: center;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    vertical-align: bottom; /* See http://stackoverflow.com/a/43266155/ */
     padding: 3px 10px;
     background-color: #fff;
     cursor: pointer;
 }
 
```

Comment:
Okay, here's a solution, but it's somewhat annoying:

Set the `.tab-switcher` to have these properties:

```css
display: flex;
justify-content: center;
align-items: center;
```

And then also manually specify the width of the tab-switcher since otherwise it will attempt to collapse inwards like an `inline-block` styled element.

It should look like this then:

<img width="578" alt="flex layout" src="https://cloud.githubusercontent.com/assets/10321399/24978652/51d52840-1f86-11e7-9d91-e6d84c49bd2f.png">